test/component/executors/testExecutorsDealing.js

```diff
@@ -48,11 +48,24 @@ describe('executors-dealing-with-estate', () => {
                 .send(sessionData)
                 .end(() => {
                     const data = {
-                        executorsApplying: 'many clouds'
+                        executorsApplying: ['many clouds']
                     };
                     testWrapper.pageUrl = testWrapper.pageToTest.constructor.getUrl();
                     testWrapper.testRedirect(done, data, expectedNextUrlForExecAlias);
                 });
         });
+
+        it('test errors message displayed for more than 4 applicants', (done) => {
```

Comment:
The validation is for the additional applicants (ie: excluding the main applicant) so perhaps this test description should be re-worded to:

`test error message displayed for more than 3 additional applicants`

Otherwise it's slightly misleading because you have 4 executors in the data.

Reply:
good point!